frontend/lindsay-wildlife-treasure-hunt-frontend/src/src/util/Session.js

```diff
@@ -14,7 +14,9 @@ export class Session {
   }
 
   static OverrideGameStateSessionData(){
-    Session.SetSessionData(GameController.gameState)
```

Comment:
whats the point of this function then? Does this break anything for production data?

Reply:
The function is only called for the dev server, so it shouldn't affect production at all. We can get rid of it as far as I'm concerned, I just was afraid to outright delete it in case I was wrong (I didn't write the function, so I was worried I was missing something).